pom.xml

```diff
@@ -147,7 +147,7 @@
     If you changes codahale.metrics.version, you also need to change
     the link to metrics.dropwizard.io in docs/monitoring.md.
     -->
-    <codahale.metrics.version>4.2.0</codahale.metrics.version>
+    <codahale.metrics.version>4.2.2</codahale.metrics.version>
```

Comment:
The URL for Dropwizard Metrics is still https://metrics.dropwizard.io/4.2.0/ so the document is not changed.